dbusadaptor.cpp

```diff
@@ -30,14 +30,11 @@ DBusAdaptor::~DBusAdaptor()
 {
```

Comment:
This comment belongs to line 25:

I'd prefer
```
DBusAdaptor::DBusAdaptor(QQuickView* view)
    : QDBusAbstractAdaptor(view)
    , m_view(view)
 {
 }
```
See other constructor initialization lists for elsewhere as well. Seems that some constructors use above style some not.